src/main/resources/k8s/def/windup.deployment.yaml

```diff
@@ -17,7 +17,7 @@ spec:
     spec:
       serviceAccountName: windup-operator
       containers:
-        - image: quay.io/windupeng/windup-operator-native:stable
+        - image: quay.io/windupeng/windup-operator-native:latest
```

Comment:
Why are all the other versions we're changing to specific release versions, except this one ?

Reply:
Well, I thought this was better to stay as "latest" as this file is intended to be used for "testing" ...... or we could change this value for each version we publish to have the specific value.
wdyt ?

Reply:
I didn't realise this was for testing purposes